ntotsc/ntotsc.tp2

```diff
@@ -210,7 +210,8 @@ LANGUAGE
 LANGUAGE
 	~Simplified Chinese(by Archenemy,Latest update by You Ye & Lewis Liu (V5.0.0))~
 	~schinese~
-	~ntotsc/language/schinese/setup.tra~
+        ~ntotsc/language/schinese/setup-%WEIDU_OS%.tra~
+	//~ntotsc/language/schinese/setup.tra~
```

Comment:
The file `setup.tra` has to be loaded as well to set all items/scripts translations for installer. Please load setup first, and then load `setup-%WEIDU_OS%.tra`.

Reply:
yes,I have done the modify.Thank you.